### DISCUSSION TITLE
REAL STUDENT GETTING REJECTED

### DISCUSSION THREAD

**Discussion by hiaercp1** [community/community#65703] [GitHub Education]
### Hi everyone,

Hi i am a real student at the high school: "Stredná priemyselná škola technická" located in Trnava Slovakia, i do not have an academic email and my request is getting rejected every single time I'm starting the studying on the September 4th this year as a new student, In every request for Student developer pack i show my ISIC card (International Student Identity Card) but i am getting rejected every single time i don't know what to do. :(
![IMG_4849](https://github.com/community/community/assets/50542095/cab8d7fa-d0e3-4e7c-9630-b8a08a4f3b91)

_No response_

**Comment by PGSCOM** [comment]
Try to send another document. In my case I send my student web portal where appears my face. Search more thinks that shows you’re an student. 

If you are getting rejected with other documents contact directly to [GitHub support](https://support.github.com/).
